ios/BT/ExposureManager.swift

```diff
@@ -489,12 +487,12 @@ private extension ExposureManager {
   func fetchExposureConfiguration() {
     apiClient.request(ExposureConfigurationRequest.get,
                       requestType: .exposureConfiguration) { [weak self] result in
-      switch result {
-      case .success(let exposureConfiguration):
-        self?.exposureConfiguration = exposureConfiguration
-      case .failure(let error):
-        print("Error fetching exposure configuration: \(error)")
-      }
+                        switch result {
+                        case .success(let exposureConfiguration):
+                          self?.exposureConfiguration = exposureConfiguration
+                        case .failure(let error):
+                          print("Error fetching exposure configuration: \(error)")
```

Comment:
Outside the scope of this pull request, but, what happens if there was an error fetching the config? Will it just keep using the previous one?

Reply:
Good question - yes, if there is an error fetching the config from the server, the hard-coded placeholder value will be used: https://github.com/Path-Check/gaen-mobile/blob/a71338f67bdfac2e57377c1b4891b58a22e133d0/ios/BT/API/Model/ExposureConfiguration.swift#L19

Reply:
Cool! I think this is good for now.

Reply:
Perhaps add a comment here to say that that's the case?